src/plugins/saved_objects_management/public/management_section/objects_table/saved_objects_table.tsx

```diff
@@ -172,13 +179,17 @@ export class SavedObjectsTable extends Component<SavedObjectsTableProps, SavedOb
 
   componentDidMount() {
     this._isMounted = true;
+
+    this.subscribleWorkspace();
```

Comment:
Is it typo here? `subscribleWorkspace`, `subscribeWorkspace`?

Reply:
good catch, fix the typo.